PlayerUI/Views/PUIPlayerView.swift

```diff
@@ -645,10 +608,12 @@ public final class PUIPlayerView: NSView {
         let b = PUIButton(frame: .zero)
 
         b.isToggle = true
-        b.image = .PUIPictureInPicture
+        b.image = AVPictureInPictureController.pictureInPictureButtonStartImage
```

Comment:
Let's use `AVPictureInPictureController.pictureInPictureButtonStartImage` for now, don't worry about deleting our asset since I have a local branch where I'm switching the player UI to SF Symbols, I'll delete all unused assets in that branch.